src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -1,42 +1,27 @@
 <?xml version="1.0"?>
 <!--
 Copyright (c) 2023 Project CHIP Authors
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <configurator>
-  <domain name="Energy Management"/>
-
+  <domain name="Energy"/>
 
-  <bitmap name="DeviceClassBitmap" type="bitmap32">
+  <bitmap name="Feature" type="bitmap32">
```

Comment:
Updated format uses features rather than a bitmap named feature.

See https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml#L58

Reply:
Device class was removed from the latest spec and most recent XML - let me make sure I have the latest xml in the PR.